tests/test_interop_run_stats_parser.py

```diff
@@ -26,5 +29,50 @@ def test_lanes_total_clusters(self):
             assert lane._total_clusters_pf is not None and lane._total_clusters_pf != 0
             assert lane._total_clusters_raw is not None and lane._total_clusters_raw != 0
 
+    def test_clusters_same_across_lanes(self):
+        """
+        Verify that 'Reads' and 'Reads Pf' are consistently the same across all reads within the same lane
+        """
+        non_index_reads = [0, 2, 3]
```

Comment:
A comment here: I realized now that the test runfolder actually only have one non-index read and it is at index 0. I don't think it affects the test, but it could be change to avoid confusion in the future. 

If you think this makes sense I would suggest fixing it in all tests except `test_interop_standardize_read_numbers` where I think it would be good to see that multiple non index reads are handled correctly.